lib/impl/AzurePresignTvm.js

```diff
@@ -43,9 +44,16 @@ class AzurePresignTvm extends Tvm {
    * @private
    */
   async _generateCredentials (params) {
+    const accountURL = `https://${params.azureStorageAccount}.blob.core.windows.net`
     const sharedKeyCredential = new azure.SharedKeyCredential(params.azureStorageAccount, params.azureStorageAccessKey)
     const containerName = Tvm._hash(params.owNamespace)
 
+    const pipeline = azure.StorageURL.newPipeline(sharedKeyCredential)
+    const serviceURL = new azure.ServiceURL(accountURL, pipeline)
+    const privateContainerURL = azure.ContainerURL.fromServiceURL(serviceURL, containerName)
```

Comment:
I see this repeated several times in `AzureBlobTvm`, `AzurePresignTvm`, `AzureRevokePresignTvm`, could you add an util like `AzureUtil.getContainerURL(sharedKeyCredential, containerName)` ?